working-groups/user-interface/About.md

```diff
@@ -0,0 +1,52 @@
+| Relevant channels on Slack | #community-website, #reporting-tool, #user-interface                                                 |
+
+## What is the goal of this working group?
+
+We are trying to follow a “sprint-like” structure where on each “sprint” cycle we start with planning and then work towards the goals defined on that plan. We're keeping track on the ongoing sprint using GitHub issues. You can know about the ongoing and upcoming work by attending our online meetings OR by looking at the agenda OR by going through the list of Github issues under the relevant repositories OR dropping a message on Slack.
+
+**The UI working group aims to accomplish the following in the coming days:**
+
+- **TSC working group notice board**. - There are several active and incubating working groups at moja global. We want to establish and develop a TSC working group notice, board. Please have a look at this [issue](https://github.com/moja-global/Google.Season.of.Documentation/issues/12) to know more.
+
+- **Community website** - The Community website aims to serve a glimpse of moja global's projects and how they work to the wider community. The website hosts projects, case studies, blogs, tutorials, and more developed by the wider moja community to help people better understand our work. It’s currently under development and all contributions towards improving the project are welcome. With the help of the UI working group, we would like to make UI components for the community website, improve accessibility and usability, and design new pages and document UI development workflow. Have a look at the [repository](https://github.com/moja-global/community-website) to know more. If you feel stuck chat with us on the [#community-website](https://app.slack.com/client/T1G1M5HPF/C022V91MY0M) channel on Slack.
+
+- **FLINT-UI** - The FLINT-UI is a web-based user interface for FLINT for configuring simulations using the FLINT.Cloud APIs and run workflows on the browser itself. The FLINT-UI is a baby project and all contributions towards making it better are welcome. You can browse this [repository](https://github.com/moja-global/FLINT-UI) to learn more about FLINT-UI.
+
+- **Reporting tool**. - The role of the reporting tool is to provide business intelligence for analyzing the FLINT output database and converting it into useful information and output. The current implementation of the reporting tool treats the Flux database as a simplified 2006 GL reporting table. We hope to refactor Angular components to be compatible with 13.x, add new features to the reporting tool, refactor old code and fix UI errors. If you want to contribute to the reporting tool, you can browse the issues in the [repository.](https://github.com/moja-global/FLINT.Reporting) If you need help chat with us on the [#reporting-tool](https://app.slack.com/client/T1G1M5HPF/C016YNZ2ALX) channel of Slack.
```

Comment:
"and converting it into useful information and output"
@chicken-biryani, could we say "into useful graphics and charts" instead? By saying "useful information" it can come across that the tool is reporting *not* useful information. :sweat_smile: